README.md

```diff
@@ -354,90 +354,132 @@ settings that machine might have that could differ from the machine used for wri
+**Quick explanation of the environment variables :**
+
+ - The environment variables named *CFP_** are use to set the connection with the database.
+ - The environment variables named *MAIL_** are use to set the mail provider. This values are not mandatory.
+ - The environment variables name *ADMIN_** are use to create the admin account in development or test environment. If an admin user already exist the initialisation processus will warn you but will keep going his initialisation. You can ignore this warn message if you already made a previus docker build.
+
+If you need to add configuration options specific to your working environment (network ...) add them to a docker-compose.override.yml file.See [this link](https://docs.docker.com/compose/extends/#understanding-multiple-compose-files) to learn more about that.
+
+Then you can build the docker images by using a [docker-compose](https://docs.docker.com/compose/install/) command which will build the images, create an admin user in development or test environment, compiling the Frontend Assets and run the containers automatically for you:
 
 ```
 $  docker-compose up
 ```
-Once the container is up and ready, go to **http://localhost:8080/** to see your server ready to use
+This command starts the following services:
+
+| Name        | Description                                                                | Port(s)            | Environment(s)                                   |
+|-------------|----------------------------------------------------------------------------|--------------------|--------------------------------------------------|
+| nodejs          | A nodejs container to compil css                                                | n/a               | all      |
+| database          | A MySQL 5.7 database server                                                | 3306               | all (prefer using a managed service in prod) |
+| php         | The OpenCfp project with PHP, PHP-FPM 7.4, Composer and sensitive configs     | n/a                | all                                              |
+| nginx       | The HTTP server for the OpenCfp project (NGINX)                               | 8080               | all                                              |
+
+<details>
+  <summary>To see the status of the containers, run:</summary>
+
+  ```bash
+  docker-compose ps
+  ```
+</details>
+
+<details>
+  <summary>To run any command in the app container you can use the docker-compose exec command:</summary>
+
+  ```bash
+  docker-compose exec <container name> <command>
+  docker-compose exec php sh # To enter the container directly, you will be placed at the root of the project
+  $ docker-compose exec php bin/console cache:clear  # To execute a cache:clear with the php console
```

Comment:
`docker-compose exec php bin/console cache:clear  # To execute a cache:clear with the php console`